fireatlas/preprocess.py

```diff
@@ -194,12 +194,11 @@ def check_preprocessed_file(
     -------
     list of unique combos of years and months (and days if NRT) that need to be processed
     """
-    fs = fsspec.filesystem(location)
```

Comment:
@lisamr: currently you are the only person using `check_preprocessed_file` so maybe we can ignore and not die on this hill 😆 but what the other preprocess functions tend to do always default to working locally and only work with `s3` if you change the `location` arg being passed in. 

If you remove all your changes in this function, then above you could get the same result by doing this. In fact, these current changes shouldn't be having `preprocessed_filename` returning any useful paths for s3 that I can see

```python
unique_ym = preprocess.check_preprocessed_file(tst, ted, sat, 'monthly', location="s3")
```

Reply:
@ranchodeluxe I'm the only one?! Well yeah, changing that location to s3 works then. 

That said, don't we want the preprocessed files in the s3 bucket, not local? If my memory is right, `check_preprocessed_file` searches for daily global VIIRS files that have been split up from the monthly one. But yeah, I don't want to die on any hill. I'm fine with your change.

Reply:
I will say that when `fs = fsspec.filesystem(None)`, it wasn't detecting the files in the shared bucket. But these changes were since `settings.fs` by default reads from s3.